iree/turbine/kernel/wave/codegen.py

```diff
@@ -1205,13 +1283,77 @@ def handle_div(lhs: Value, rhs: Value) -> OpResult:
     return result
 
 
+@handle_binary_op(operator.gt)
+def handle_gt(lhs: Value, rhs: Value) -> OpResult:
+    element_type = get_type_or_element_type(lhs.type)
+    if _is_float_type(element_type):
+        result = arith_d.cmpi(arith_d.CmpFPredicate.OGT, lhs, rhs)
+    elif _is_integer_like_type(element_type) and (
+        element_type.is_signed or element_type.is_signless
+    ):
+        result = arith_d.cmpi(arith_d.CmpIPredicate.sgt, lhs, rhs)
+    elif _is_integer_like_type(element_type) and element_type.is_unsigned():
```

Comment:
Note that `is_unsigned` is a property, it should not be called. Here and elsewhere.

Reply:
good catch, thanks! :)